app/src/main/java/com/willowtree/vocable/settings/EditCategoryOptionsFragment.kt

```diff
@@ -176,23 +181,30 @@ class EditCategoryOptionsFragment : BaseFragment<FragmentEditCategoryOptionsBind
     }
 
     private fun handlePhrases(phrases: List<Phrase>) {
-        with(binding.editCategoryPhraseHolder) {
-            isSaveEnabled = false
-
-            adapter = phrasesAdapter
-
-            phrasesAdapter.setItems(phrases)
-
-            // Move adapter to middle so user can scroll both directions
-            val middle = phrasesAdapter.itemCount / 2
-            if (middle % phrasesAdapter.numPages == 0) {
-                setCurrentItem(middle, false)
-            } else {
-                val mod = middle % phrasesAdapter.numPages
-                setCurrentItem(
-                    middle + (phrasesAdapter.numPages - mod),
-                    false
-                )
+        if (phrases.isEmpty()) {
+            binding.emptyPhrasesText.visibility = View.VISIBLE
+            binding.emptyAddPhraseButton.visibility = View.VISIBLE
+        } else {
+            binding.emptyPhrasesText.visibility = View.GONE
+            binding.emptyAddPhraseButton.visibility = View.GONE
```

Comment:
There's actually a Kotlin Android extension to help with toggling between `VISIBLE` and `GONE`:
`binding.emptyPhrasesText.isVisible = true`

Reply:
🙏